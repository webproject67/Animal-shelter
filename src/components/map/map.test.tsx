import * as React from "react";
import * as renderer from "react-test-renderer";
import Map from "./map";

it(`Render Map`, () => {
  const root = renderer.create(<Map 
    src={`1`}
    onButtonClick={() => {}}
  />).toJSON();
 
  expect(root).toMatchSnapshot();
});