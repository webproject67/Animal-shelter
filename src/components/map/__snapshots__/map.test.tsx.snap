// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Map 1`] = `
<div
  className="modal-content"
>
  <button
    aria-label="Ð¡ancel"
    className="modal-content__cancel"
    onClick={[Function]}
  >
    <svg
      className="modal-content__icon"
    >
      <use
        xlinkHref="img/svg/sprite.svg#cancel"
      />
    </svg>
  </button>
  <div
    className="modal-content__wrapper"
  >
    <iframe
      aria-hidden="false"
      frameBorder="0"
      height="500"
      src="1"
      style={
        Object {
          "border": 0,
        }
      }
      width="1000"
    />
  </div>
</div>
`;
